docs/aws.md

```diff
@@ -540,6 +540,7 @@ and the Batch Execution Role.
 
 :::{note}
 This feature requires the use {ref}`Wave <wave-page>` container provisioning service.
+This is because `s5cmd` is used to stage and unstage data which is augmented into the task container via wave.
 :::
```

Comment:
Suggested change by @pditommaso (some minor tweaks by me):
```suggestion
:::{note}
Nextflow uses the [s5cmd](https://github.com/peak/s5cmd) to download the task input data and upload the task outputs.
To allow this capability in your pipeline, you need to enable the Wave service in the Nextflow configuration, as shown in the above example. See {ref}Wave <wave-page> documentation for more details.
:::
```

That ok @christopher-hakkaart ?

Reply:
Committing so you can comment better